docs/tutorials/first_app/blinky_compiler.md

```diff
@@ -1,70 +1,81 @@
+ 
+<span class="images">![](https://sarahmarshy.github.io/img/import-compiler.png)
+</span>
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import_dialog.png)<span>Importing Blinky</span></span>
+#### Compile
+Click on the "Compile" button, your browser will then download the program as a executable file.
 
-The import mechanism offers a default name, but you're free to change it. When you're done, click **Import**.
+<span class="images">![](https://sarahmarshy.github.io/img/compile.png)
+</span>
 
-### Viewing Blinky
+#### Program
+Open the folder where the executable file was downloaded, then click and drag (or copy and paste) the file to your mbed board's USB device folder.
 
-The imported Blinky has two interesting parts:
+Once the file has been flashed to the board, press the board's "reset" button and you should now see the LED blinking.
 
-- `main.cpp`, where the Blinky-specific code is. You can double-click the file in the navigation pane on the left to view the code.
-- `mbed-os`, where the Arm Mbed OS codebase is.
 
-Later we'll compile the code; this will take both of these parts and create a single application file from them.
+### Debug
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/maincpp.png)<span>Viewing the code in main.cpp</span></span>
+#### Desktop IDE
 
-### Selecting a target board
+To debug using a desktop IDE such as Keil uVision, IAR, or Eclipse, click the "Export" button under "Program Details", select your export platform and IDE and click "Export". Your browser will then download a zip file with the project files.
 
-The Mbed Online Compiler can build your application to match any Arm Mbed Enabled board. However, you have to select the target board before compiling.
+<span class="images">![](https://sarahmarshy.github.io/img/export.png)
+</span>
 
-#### Adding a board to your list
+#### Printf
 
-To add a board to your list, go to <a href="https://os.mbed.com/platforms/" target="_blank">the board's page on `os.mbed.com`</a>, and click the **`Add to your Mbed Compiler`** button:
+Another way to do basic debugging is to use the `printf` command in your code, then read the output using a serial terminal such as [PuTTY](http://www.putty.org/) or [CoolTerm](http://freeware.the-meiers.org/). For example, add `printf("Hello World!\n\r");` to the top of your main function, then recompile the program and flash it to your device.
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/add_board.png)<span>Adding a board to the Mbed Online Compiler's board list</span></span>
+Unless otherwise specified, `printf` defaults to a baud rate/speed of `9600` on mbed OS. To determine which communication port your board is connected to, follow the instructions for your operating system below:
 
-#### Selecting a board
+##### Windows
```

Comment:
This section feels out of place. In the offline section, we listed environment first. Here, we list it last. Also, I don't think we should talk about using the command-line in the section about the Mbed Online Compiler.

Reply:
Im not sure I understand what you are trying to say, lets discuss in person